README.md

```diff
@@ -3,6 +3,10 @@ This repository is intended to centralize string data common to many builds
 (such as required compiler switches), both internal to Linden and external, on
 TeamCity and on individual dev machines.
 
+This fork holds data for viewer (and viewer library) builds. Changes here are
+viewer-only. To affect both viewer and server, push to
+https://bitbucket.org/lindenlab/build-variables and pull the changes.
```

Comment:
hmm, reference to "this fork" dates back to when there were `build-variables`, `viewer-build-variables` and `server-build-variables`. I'll push a change to delete this paragraph.